### ISSUE TITLE
Cannot launch some content with command line

### ISSUE THREAD

**Issue by lasers** [libretro/RetroArch#4233]
I'm not able to launch some content with command line. No good.

* `dinothawr_libretro.so` --> `dinothawr.game`
* `lutro_libretro.so` --> `Platformer.lutro`

I tested others. All good.

* `2048_libretro.so` --> `(none)`
* `nxengine_libretro.so` --> `Doukutsu.exe`
* `gw_libretro.so` --> `Donkey Kong Jr. (Coleco).mgw`
* `prboom_libretro.so` --> `doom1.wad`

**Comment by fr500** [comment]
logs?

**Comment by lasers** [comment]
Dinothawr and Lutro both printed same thing.
```
RetroArch [INFO] :: Found joypad driver: "udev".
RetroArch [INFO] :: Using font rendering backend: freetype.
RetroArch [INFO] :: Starting threaded audio driver ...
RetroArch [INFO] :: [Audio Thread]: Initializing audio driver.
RetroArch [ERROR] :: Cannot open threaded audio driver ... Exiting ...
RetroArch [ERROR] :: Fatal error received in: "audio_driver_init_internal()"
```

**Comment by RobLoach** [comment]
The following worked for me...

```
~/Documents/RetroArch/retroarch -L /usr/lib/libretro/dinothawr_libretro.so ~/Games/Dinothawr-master/dinothawr/dinothawr.game
```

Looks like your audio driver fails to initiate?

**Comment by lasers** [comment]
Yes. Is it something I have to do on my side or something to be made on RetroArch side? I can load Dinothawr and Lutro okay under XMB. I'm not sure if this is related, but under XMB, I will have to "Run" twice sometimes to get it going. I tried launching it under CLI few more times to see if I'm able to load it fine. Hmm, nothing.

**Comment by lasers** [comment]
Both works fine after switching to Pulseaudio.

**Comment by Alcaro** [comment]
No, this bug is still around. The solution to "RetroArch doesn't work on Windows" isn't "switch to Linux", it's making it work on Windows.

Same applies to "RetroArch doesn't work on ALSA" (or whatever driver you tried).

**Comment by fr500** [comment]
I think these issues may be due to the cores themselves.
I know dinothawr at least has an overengineered mixer

**Comment by zoltanvb** [comment]
Tested with alsa, both Lutro and Dinothawr launch fine nowadays. Probably fixed somewhere along the way.
